tox.ini

```diff
@@ -31,10 +35,21 @@ usedevelop = True
 
 [testenv:metadata-validation]
 deps =
-    collective.checkdocs
-    twine
+    collective.checkdocs>=0.2
+    pep517 >= 0.5.0
+    pip>=19.1.1
+    setuptools>=39.0
+    twine>=1.13.0
+    wheel>=0.31.0
+
 usedevelop = False
 # Ref: https://twitter.com/di_codes/status/1044358639081975813
 commands =
-    python -m setup checkdocs check --metadata --restructuredtext --strict --verbose
-    twine check .tox/dist/*
+    rm -rf {toxinidir}/dist
+    python -m pep517.build \
```

Comment:
1) this should be a separate testenv, you can then reuse parts of it in other places. also, you can specify deps between testenvs.
2) this is achievable by using `isolated_build = true` so not sure why you want to invoke it explicitly.